crates/iota-crypto/Cargo.toml

```diff
@@ -104,6 +104,7 @@ wasm-bindgen-test = "0.3"
 # Since both versions are in our dependency tree, we need to overwrite them with the proper feature
 getrandom_2 = { version = "0.2", package = "getrandom", features = ["js"] }
 getrandom_3 = { version = "0.3", package = "getrandom", features = ["wasm_js"] }
+test-strategy = "=0.4.1"
```

Comment:
The import in the [dev-dependencies] should be sufficient, no?

Reply:
This is a fix for an issue specific to wasm, but I don't really understand it, I just took it from upstream